src/gax/src/lib.rs

```diff
@@ -54,3 +54,7 @@ pub mod polling_backoff_policy;
 pub mod polling_error_policy;
 pub mod retry_policy;
 pub mod retry_throttler;
+
+#[cfg(feature = "unstable-sdk-client")]
+#[doc(hidden)]
+pub mod retry_loop;
```

Comment:
optional: say `internal` somewhere in addition to hiding this from the docs?

Reply:
Also FWIW, `unstable-sdk-client` should already be hidden from the docs. It doesn't hurt to also have `#[doc(hidden)]` though

Reply:
Ack on the documentation.
I changed the name to `retry_loop_internal`. This may belong on a separate crate, but I would like to finish all the refactoring around `auth` before figuring that out.